static/faq.html

```diff
@@ -71,6 +71,7 @@ <h2 id="table-of-contents">
 <!--Content below -->
 
 <!--Site footer below. Do not edit below this line.--> 
+        </div>
```

Comment:
It seems you accidentally added a div here. Also, can you squash these into a single commit? Use `git rebase -i PARENT_COMMIT` (the one before your first commit) and then switch this 2nd commit from pick to squash. Do not reorder the lines, as that reorders commits.

Reply:
Whoops. On it now.